pyspades/player.py

```diff
@@ -802,7 +802,7 @@ def refill(self, local: bool = False) -> None:
         if not local:
             restock = loaders.Restock()
             self.send_contained(restock)
-            if self.client_info and self.client_info["client"] == "Ace of Spades" or not self.client_info:
+            if self.client_info and self.client_info.get("client", "Ace of Spades") == "Ace of Spades" or not self.client_info:
```

Comment:
I don't think we actually have a case where `self.client_info` isn't a dict, I don't see any place where it's set to `None`, so this could be shortened to just `self.client_info.get("client", "Ace of Spades") == "Ace of Spades"`.